TrackletAlgorithm/TrackHandler.h

```diff
@@ -0,0 +1,50 @@
+#ifndef TrackletAlgorithm_TrackHandler_h
+#define TrackletAlgorithm_TrackHandler_h
+#include "TrackFitMemory.h"
+#include "hls_stream.h"
+const int NBarrelStub = 4;
+const int NDiskStub = 4;
+const unsigned int kFullBarrelStubSize = TrackFit<NBarrelStub, NDiskStub>::kBarrelStubSize * 4;
+const unsigned int kFullDiskStubSize = TrackFit<NBarrelStub, NDiskStub>::kDiskStubSize * 4;
+const unsigned int kFullTrackWordSize = TrackFit<NBarrelStub, NDiskStub>::kTrackWordSize;
+const unsigned int kBarrelStubIndexSizeMSB = TrackFit<NBarrelStub, NDiskStub>::kTFBarrelStubRSize + TrackFit<NBarrelStub, NDiskStub>::kTFPhiResidSize + TrackFit<NBarrelStub, NDiskStub>::kTFZResidSize + TrackFit<NBarrelStub, NDiskStub>::kTFStubIndexSize;
+const unsigned int kBarrelStubIndexSizeLSB = TrackFit<NBarrelStub, NDiskStub>::kTFBarrelStubRSize + TrackFit<NBarrelStub, NDiskStub>::kTFPhiResidSize + TrackFit<NBarrelStub, NDiskStub>::kTFZResidSize;
+const unsigned int kDiskStubIndexSizeMSB = TrackFit<NBarrelStub, NDiskStub>::kTFDiskStubRSize + TrackFit<NBarrelStub, NDiskStub>::kTFPhiResidSize + TrackFit<NBarrelStub, NDiskStub>::kTFRResidSize + TrackFit<NBarrelStub, NDiskStub>::kTFStubIndexSize;
+const unsigned int kDiskStubIndexSizeLSB = TrackFit<NBarrelStub, NDiskStub>::kTFDiskStubRSize + TrackFit<NBarrelStub, NDiskStub>::kTFPhiResidSize + TrackFit<NBarrelStub, NDiskStub>::kTFRResidSize;
+const unsigned int layerStubIndexSize = 1; // 7 stubs per layer
+
+struct TrackStruct {
+  
+  TrackFit<NBarrelStub, NDiskStub>::TrackWord _trackWord = 0; 
+  TrackFit<NBarrelStub, NDiskStub>::BarrelStubWord _barrelStubArray[4][layerStubIndexSize] = {0}; 
+  TrackFit<NBarrelStub, NDiskStub>::DiskStubWord _diskStubArray[4][layerStubIndexSize] = {0};
```

Comment:
Again, please use `NBarrelStub` and `NDiskStub` instead of the magic number `4`.